cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
It was not meant to be readable but informative for enlightened individuals :)
 
The latest commit restores the ms since last line display. It could also display ms since start, but I would guess every time you actually want to use that information it's the duration between events you are interested in, as you get the total execution time in the at_exit hook anyway.